open-apparel-registry/open-apparel-registry
# Add a function for parsing uploaded CSV line items

## Overview

The new `parse_facility_list_item` function will eventually be run in a background task to process uploaded CSV file lines stored as `FacilityListItem` objects.

Connects #76

## Notes

I added arguments to `scripts/test` in order to run pieces of the system test suite, allowing for faster development iteration and a more comfortable test-first workflow.

## Testing Instructions

* Verify that `./scripts/test --django` completes without error.
* Verify that `./scripts/test` runs all linters and tests`
* Verify the other test flags
  * `./scripts/test --lint`
  * `./scripts/test --app`
